pyproject.toml

```diff
@@ -31,3 +27,34 @@ module = [
   "paramiko.*",
 ]
 ignore_missing_imports = true
+
```

Comment:
We should remove the `[tool.black]` section too, correct?

Reply:
Removed.